testhelper/tests.go

```diff
@@ -228,6 +236,8 @@ func (options *TestOptions) testTearDown() {
 				logger.Log(options.Testing, "Deleting the temp working directory")
 				os.RemoveAll(options.baseTempWorkingDir)
 			}
+			//Clean up terraform files
+			CleanTerraformDir(options.TerraformDir)
```

Comment:
Will this be an issue if the TerraformDir was a temp dir that was removed a few lines up?

Reply:
No its best effort, it will log an error but wont fail anything